starters/features/vitest/vitest.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'vite';
+import { qwikVite } from '@builder.io/qwik/optimizer';
+import { qwikCity } from '@builder.io/qwik-city/vite';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+export default defineConfig(() => {
+  return {
+    plugins: [qwikCity(), qwikVite(), tsconfigPaths()],
+    define: {
```

Comment:
So, we should make our qwikVite() plugin to be aware of vitest, i think vitest sets the mode as "test" or "benchmark", so we can detect that and set the flags automatically.

Having a separate vitest.config.ts will introduce other problems since it introduces one more chore for developers to keep it in sync with `vite.config.ts`

So i would remove `vitest.config.ts` and make our Qwik vite plugin be aware of testing